src/inventory/mod.rs

```diff
@@ -472,9 +489,10 @@ impl Item {
     }
 }
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Default)]
 pub enum Material {
-    Air,                             // 1.7.10 (id: 0, stack: 0)| 1.13 (id: 9648)
+    #[default]
+    Air, // 1.7.10 (id: 0, stack: 0)| 1.13 (id: 9648)
```

Comment:
could you shift this comment back?

Reply:
I tried doing this already, but for some reason `cargo fmt` shifts this one line forward every time it is run.